### DISCUSSION TITLE
Get parent node via API

### DISCUSSION THREAD

**Discussion by Frankwii** [nvim-tree/nvim-tree.lua#2858] [Q&A]
Hi, I'm kind of new to nvim-tree and nvim in general, so perhaps this is a stupid question.

Is it possible to get the parent of a given node via API? I can get the "current" node (the one under the cursor) via api.tree.get_node_under_cursor(), but I would like to get the parent of that node. How can I achieve this?

Maybe the "node" object has some atribute storing the parent node? (Assuming Lua object-oriented; I'm really not sure)

Thanks

**Comment by alex-courtis** [comment]
It's `parent`.

Try this:
```lua
    local node = api.tree.get_node_under_cursor()
    print(vim.inspect(node.parent, { depth = 3 }))
```
